### ISSUE TITLE
Additional filters for 'space_member_usage'.

### ISSUE THREAD

**Issue by kputschko** [rstudio/rscloud#49]
In the help page for the `space_member_usage` function, I see there is a filters argument that makes use of both `groupby` and `from`.  I'm wondering if there are any more parameters that could be passed into this filter argument?

**Comment by kputschko** [comment]
I'm running into another issues as of today.  This code ran successfully last week.

```
space_member_usage(space, filters = list(groupby = "user_id", from = "90d"))
```

But now it is returning this error:

```
Error: Problem with `mutate()` column `last_activity`.
i `last_activity = as.POSIXct(.data$last_activity/1000, origin = "1970-01-01")`.
x Column `last_activity` not found in `.data`
```

Interestingly, I can use `from = 30`, but not `from = 35`.

```
# Success
space_member_usage(space, filters = list(groupby = "user_id", from = "30d"))

# Failure
space_member_usage(space, filters = list(groupby = "user_id", from = "35d"))
```

**Comment by mine-cetinkaya-rundel** [comment]
@kputschko There was an update in the last week in the API to limit `last_activity` to 30 days. I'm working on updating the function to reflect this change so the function works for any time period but reports `last_activity` only up to 30 days and for time periods longer than that it doesn't report it.

**Comment by kputschko** [comment]
Thanks for the response. To clarify, going forward it will only be possible to look back at time frames within the last 30 days?

**Comment by mine-cetinkaya-rundel** [comment]
Yes, for the `last_active` field it will only be possible to look back within the last 30 days. For other fields there won't be such time limit.

**Comment by mine-cetinkaya-rundel** [comment]
@kputschko The broken `space_member_usage()` call for longer time periods should now be fixed with #50. Leaving the issue open to remind of the original request re: better documentation of possible filters.
